packages/orbit-components/src/common/placements.ts

```diff
@@ -0,0 +1,39 @@
+import type { Placement as FloatingPlacement, Alignment } from "@floating-ui/react";
+
+export type AutoPlacement = "auto" | "auto-start" | "auto-end";
+export type Placement = FloatingPlacement;
+
+export enum AUTO_PLACEMENTS {
+  AUTO = "auto",
+  AUTO_START = "auto-start",
+  AUTO_END = "auto-end",
+}
+
+export enum PLACEMENTS {
+  TOP = "top",
+  TOP_START = "top-start",
+  TOP_END = "top-end",
+  BOTTOM = "bottom",
+  BOTTOM_START = "bottom-start",
+  BOTTOM_END = "bottom-end",
+  RIGHT = "right",
+  RIGHT_START = "right-start",
+  RIGHT_END = "right-end",
+  LEFT = "left",
+  LEFT_START = "left-start",
+  LEFT_END = "left-end",
+}
+export const isFixedPlacement = (placement: string): placement is Placement =>
```

Comment:
```suggestion
export const isFixedPlacement = (placement: string): placement is FixedPlacement =>
```

Reply:
I wonder if renaming this to FixedPlacement would still make sense and be more readable (given the name of the function)?

Reply:
Yes, this was done 🙂